### PR TITLE
Generalization in annotator, helps and use of the plugin

### PR DESCRIPTION
**Pull request by Vilax**

### CODE REVIEW

**Review by JorMaister — APPROVED** [review]

File: tomosegmemtv/protocols/protocol_annotate_membranes.py [hunk 1]
```diff
@@ -94,17 +151,31 @@ def _summary(self):
...
+                error.append('The sampling rate of the tomograms does not match the sampling rate of the input masks')
+
+        return error
+
     # --------------------------- UTIL functions -----------------------------------
 
     def _initialize(self):
-        self._tomoList = [tomo.clone() for tomo in self.inputTomoMasks.get().iterItems()]
-        self._provider = MembAnnotatorProvider(self._tomoList, self._getExtraPath(), 'membAnnotator')
+        import time
```
Comment (pconesa) [inline tomosegmemtv/protocols/protocol_annotate_membranes.py:167]: I think we do not want this delay?

File: tomosegmemtv/protocols/protocol_annotate_membranes.py [hunk 2]
```diff
@@ -94,17 +151,31 @@ def _summary(self):
...
+        time.sleep(12)
+        self._tomoMaskDict = {tomoMask.getTsId(): tomoMask.clone() for tomoMask in self.inputTomoMasks.get().iterItems()}
+        if self.inputTomos.get():
+            self._tomoDict = {tomo.getTsId(): tomo.clone() for tomo in self.inputTomos.get().iterItems()}
```
Comment (pconesa) [inline tomosegmemtv/protocols/protocol_annotate_membranes.py:171]: Where is _tomoDict used?

File: tomosegmemtv/protocols/protocol_tomosegmentv.py [hunk 3]
```diff
@@ -51,7 +51,35 @@ class outputObjects(Enum):
 
 
 class ProtTomoSegmenTV(EMProtocol):
-    """Segment membranes in tomograms"""
+    """TomoSegMemTV is a software suite for segmenting membranes in tomograms. The method
```
Comment (pconesa) [inline tomosegmemtv/protocols/protocol_tomosegmentv.py:54]: Wow! Very detailed.

File: tomosegmemtv/protocols/protocol_annotate_membranes.py (hunk 1 above)
Comment (Vilax) [reply tomosegmemtv/protocols/protocol_annotate_membranes.py]: We do not, solved

File: tomosegmemtv/protocols/protocol_annotate_membranes.py (hunk 2 above)
Comment (Vilax) [reply tomosegmemtv/protocols/protocol_annotate_membranes.py:169]: It is used in the input, to represent the tomograms by the annotator. It is used for visualization purpose

File: CHANGES.txt [hunk 4]
```diff
@@ -1,3 +1,6 @@
+v3.2.0
```
Comment (pconesa) [inline CHANGES.txt:1]: Can we bump the version in the __init__ and release this?

**Pull request merged**